lib/kitchen/driver/ec2.rb

```diff
@@ -98,10 +98,10 @@ class Ec2 < Kitchen::Driver::SSHBase
         end
       end
 
-      def self.iam_creds
+      def iam_creds
         @iam_creds ||= begin
-          fetch_credentials(use_iam_profile:true)
-        rescue RuntimeError => e
+          fetch_credentials(use_iam_profile: true)
+        rescue RuntimeError, NoMethodError => e
```

Comment:
https://github.com/fog/fog-aws/blob/master/lib/fog/aws/credential_fetcher.rb#L27

fetch_credentials will always call super if it fails